docs/users.md

```diff
@@ -0,0 +1,78 @@
+# Users
+
+This is a collection of user methods. 
+
+## ToC
+
+- [`sdk.users.me()`](#me)
+- [`sdk.users.get(id)`](#getById)
+- [`sdk.users.lookup(email)`](#lookByEmail)
+
+<a id="me"></a>
+
+## `sdk.users.me()`
+This method is used to get details about the current logged in user.
+
+### API
+```js
+sdk.users.me(): Promise<User>
+```
+
+### Example
+
+```js
+const eventbrite = require('eventbrite');
+
+// Create configured Eventbrite SDK
+const sdk = eventbrite({token: 'OATH_TOKEN_HERE'});
+
+sdk.users.me().then((user) => {
+    console.log(`Hi ${user.name}!`);
+});
+```
+
+<a id="getById"></a>
+
+## `sdk.users.get(id)`
+This method is used to load the details for a specific user by their user id. 
+
+### API
+```js
+sdk.users.get(id: number|string): Promise<User>
```

Comment:
This can only take a string. JavaScript numbers aren't big enough to fully describe our user IDs

Reply:
I was looking based on the sample code and wasn't sure I went board to start. Updating to just string.